samples/dnn/text_detection.cpp

```diff
@@ -5,24 +5,28 @@
 using namespace cv;
 using namespace cv::dnn;
 
+
 const char* keys =
-    "{ help  h     | | Print help message. }"
-    "{ input i     | | Path to input image or video file. Skip this argument to capture frames from a camera.}"
-    "{ model m     | | Path to a binary .pb file contains trained network.}"
-    "{ width       | 320 | Preprocess input image by resizing to a specific width. It should be multiple by 32. }"
-    "{ height      | 320 | Preprocess input image by resizing to a specific height. It should be multiple by 32. }"
-    "{ thr         | 0.5 | Confidence threshold. }"
-    "{ nms         | 0.4 | Non-maximum suppression threshold. }";
+        "{ help  h           | | Print help message. }"
+        "{ input i           | | Path to input image or video file. Skip this argument to capture frames from a camera.}"
+        "{ model_decoder m   | | Path to a binary .pb file contains trained detector network.}"
+        "{ model_recognition | | Path to a binary .pb file contains trained recognition network.}"
```

Comment:
helo, just curious, which model is used here ? 

and if it is not (yet) in the [model zoo](https://github.com/opencv/opencv_extra/blob/master/testdata/dnn/download_models.py), shouldn't there be a download link somewhere ?

Reply:
@rngtna Please add a link to trained model and code in comments of how to convert it to ONNX.